src/components/common/ContactInfo.vue

```diff
@@ -0,0 +1,75 @@
+<template>
```

Comment:
If you have time, consider moving this to bcrs-shared-components.

Reply:
I'm definitely over time on this ticket 😓